types/mongodb/test/collection/findX.ts

```diff
@@ -24,6 +24,22 @@ async function run() {
   });
   const res: Cursor<TestModel> = collectionT.find({});
 
+  await collectionT.findOne({}, {
+    projection: {
+    },
+    sort: {}
+  });
+
+  await collectionT.findOne({}, {
+    projection: {
+      stringField: {$meta: 'textScore'},
+      fruitTags: {$min: 'fruitTags'},
+      max: {$max: ['$max', 0]},
+    },
+    sort: {stringField: -1, text: {$meta: 'textScore'}, notExistingField: -1}
```

Comment:
Shouldn't TS complain here about the `notExistingField`?

Reply:
No, because you may use dot notation to access nested object like `obj.prop`, the TS only can treat it as a string, so `notExistingField` is not a valid key but TS can't recognize.